# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 2424d5c30267
# Full Hash: 2424d5c30267f67ab2b28119d75d89f0441b30e0
# Author: Gijs Kruitbosch <gijskruitbosch@gmail.com>
# Date: 2024-01-23 05:36:48
# Regressor Bug: 1833814
# File Overlap Count: 1
# Description:
#   Bug 1833814 - change nsIWidget's cursor logic, r=emilio,win-reviewers,mhowell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195549
# ==============================================================================

diff -r 94f46aad00df -r 2424d5c30267 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon Jan 22 20:10:54 2024 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon Jan 22 20:42:50 2024 +0000
@@ -8844,6 +8844,11 @@
   value: 32
   mirror: always
 
+- name: layout.cursor.disable-for-popups
+  type: bool
+  value: true
+  mirror: always
+
 - name: layout.display-list.build-twice
   type: RelaxedAtomicBool
   value: false